osidb/mixins.py

```diff
@@ -809,6 +805,10 @@ def validate(self, raise_validation_error=True, dry_run=False):
                         **(e.params or {}),
                     )
 
+        # standard validations
+        # exclude meta attributes
+        self.full_clean(exclude=["meta_attr"])
```

Comment:
This change should be done in a separate commit as it does not directly relate to PS components or PURLs but with the fact that our custom validations should get priority before the standard ones (as they are more specific and provide better readable and more targeted error messages).